### PR TITLE
OCPBUGSM-10683 Deploy UI: pull image

### PR DESCRIPTION
**Pull request by RazRegev**
Try to pull ui image locally before using it to create deployment configuration.

### CODE REVIEW

File: tools/deploy_ui.py [hunk 1]
```diff
@@ -14,7 +14,9 @@ def main():
 
     dst_file = os.path.join(os.getcwd(), "build/deploy_ui.yaml")
     image_fqdn = deployment_options.get_image_override(deploy_options, "ocp-metal-ui", "UI_IMAGE")
-    cmd = '{command} run {image} /deploy/deploy_config.sh -i {image} -n {namespace}'.format(command=utils.get_runtime_command(), image=image_fqdn, namespace=deploy_options.namespace)
+    runtime_cmd = utils.get_runtime_command()
+    utils.check_output(f'{runtime_cmd} pull {image_fqdn}')
```
Comment (filanov) [inline tools/deploy_ui.py:18]: Does podman pull the image if it's already exist?

Comment (RazRegev) [reply tools/deploy_ui.py:18]: when the image is updated yes

Comment (filanov) [reply tools/deploy_ui.py:18]: I think that we had an issue with it and podman, @tsorya do you remember something like this?

Comment (tsorya) [reply tools/deploy_ui.py:18]: nor podman nor docker pulls image if it already exists. 
U can set flag the will do it

Comment (tsorya) [reply tools/deploy_ui.py:18]: not relevant comment : this is pull command, it will always try to pull

**Pull request merged**